x/staking/types/keys.go

```diff
@@ -40,6 +40,7 @@ var (
 	RedelegationKey                  = []byte{0x34} // key for a redelegation
 	RedelegationByValSrcIndexKey     = []byte{0x35} // prefix for each key for an redelegation, by source validator operator
 	RedelegationByValDstIndexKey     = []byte{0x36} // prefix for each key for an redelegation, by destination validator operator
+	DelegationByValIndexKey          = []byte{0x37} // key for delegations by a validator
```

Comment:
Sorry to poke my head here, but this is a bug no? Looks like the same key value as `UnbondingIDKey` below?

Reply:
yup that is right, cant wait to rewrite this module to use colletions. @atheeshp can you fix this please

Reply:
Sorry, haven't noticed it. Raised fix PR(#16176) thanks for finding it :pray: .

Reply:
@tac0turtle has this PR been backported to SDK47?

Reply:
This feature isn't included in v0.47, but is in v0.50.